File: lab5java/exercise2/Driver.java
package lab5java.exercise2;

public class Driver {
    public static void main(String[] args) {
        Vehicle car = new Vehicle(2, 6, "blue", 14.6, 3);
        Taxi cab = new Taxi(3.3);
        Truck mack = new Truck(7.54);

        mack.setCargo(true);

        car.print();
        cab.print();
        mack.print();
    }
}

File: lab6/driver.cpp
// William ANDERSON 15494372

#include <iostream>

using std::cout;
using std::endl;

#include <vector>

using std::vector;

#include "vehicle.h"
#include "taxi.h"
#include "truck.h"

int main()
{
   Taxi cab( 3.3 );
   Truck mack( 7.54 );





   parkingLot[ 0 ] = &cab;
   parkingLot[ 1 ] = &mack;

   cout << "\nThe vehicles cannot get out of their parking spaces because of "
        << "traffic,\nso they respond: \n";

   for ( int i = 0; i < parkingLot.size(); i++ ) {



      cout << endl;
      cout << endl;
   }
   return 0;
}
